### PR TITLE
update elasticsearch support

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

See https://camunda.slack.com/archives/C026U8GBNSW/p1698345189792509 and https://camunda.slack.com/archives/C026U8GBNSW/p1698393211411889.

@zszakolci We have the following in the upgrade guide, but is there a way you would like to see this more visible?

Screenshot 2023-10-27 at 8 20 44 AM

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by alexcorremans** [comment]
Hi @christinaausley, responding for @zszakolci - yes, we would like to see something on [the 8.2 to 8.3 upgrade guide](https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/820-to-830/) that says that Elasticsearch 7 is no longer supported in 8.3, only Elasticsearch 8.8 and higher. Currently on that page there is only a link to the [Helm chart upgrade guide](https://docs.camunda.io/docs/self-managed/platform-deployment/helm-kubernetes/upgrade/#v830-minor) which says that Elasticsearch was 'upgraded' from v7.x to v8.x. 

In other words:
* this should be in the main upgrade guide, not just the Helm chart guide, so it's more visible
* it should be clear that v7 is no longer supported, not just that it was upgraded in the Helm chart
* it should be clear that only v8.8 and higher are supported (not v8.x)

Thanks!

**Comment by christinaausley** [comment]
This is ready for final review 👍

**Review by akeller — COMMENTED** [review]
This is a bit of a nitpick, but I would remove "currently" from the phrasing. However, if this helps Support, I'm ok with leaving it.

"Currently" should be implied with where the documentation exists in our versioning.

**Comment by christinaausley** [comment]
@kuchenmn Do you have any additional comments here?

**Comment by kuchenmn** [comment]
> @kuchenmn Do you have any additional comments here?

Are there updates to the Manual Installation docs page? - https://docs.camunda.io/docs/self-managed/platform-deployment/manual/#download-and-run-elasticsearch

**Comment by christinaausley** [comment]
> > @kuchenmn Do you have any additional comments here?
> 
> Are there updates to the Manual Installation docs page? - https://docs.camunda.io/docs/self-managed/platform-deployment/manual/#download-and-run-elasticsearch

We do currently have an admonition:

:::note
Camunda is currently compatible with Elasticsearch 8.8.x (see [supported environments](/docs/reference/supported-environments.md)) which you can [download here](https://www.elastic.co/downloads/past-releases/elasticsearch-8-8-0).
:::

**Review by akeller — APPROVED** [review]
I'm good with these changes. Given that support is involved, we should probably merge and release to make this content available.

**Comment by kuchenmn** [comment]
Perfect, thank you. Nothing else from my side.

**Review by alexcorremans — COMMENTED** [review]
I'm also in favour of leaving 'currently' out. Ideally I'd like to see something like 'and no longer supports Elasticsearch 7.x', is there a reason why we don't want to be explicit about that?

**Comment by akeller** [comment]
> I'm also in favour of leaving 'currently' out. Ideally I'd like to see something like 'and no longer supports Elasticsearch 7.x', is there a reason why we don't want to be explicit about that?

@alexcorremans In general, we want to document what is possible only. It becomes complex quickly if we try to document both. 

Is this not clear enough via the [Supported Environments](https://docs.camunda.io/docs/reference/supported-environments/)? We can add something to the Announcements page about end of support for ES 7 if this is causing thrash.

**Comment by alexcorremans** [comment]
Ok, I thought we could document changes likes this under the upgrade guides. If that's not our policy, then something under Announcements would be great, thank you. We do have customers who did not realize ES 7.x was no longer supported.

**Review by akeller — APPROVED** [review]
Thank you for the quick iterations here @christinaausley!

**Comment by alexcorremans** [comment]
Thank you @christinaausley ! Can that note in Announcements go under the Camunda 8.3 header and above the 'Versioning changes in Helm chart' section? The Helm chart changes happened starting from 8.2.8 whereas this is about a change from 8.3 onwards.

**Comment by christinaausley** [comment]
@alexcorremans Sure thing! I'll launch a PR today to clean it up.
